Tasks/VsTestPlatformToolInstallerV1/nugetpackageversionhelper.ts

```diff
@@ -18,15 +18,19 @@ export class NugetPackageVersionHelper {
 
         this.consolidatedCiData.includePreRelease = `${includePreRelease}`;
 
-        nugetTool.arg(constants.list).arg(`packageid:${constants.packageId}`).argIf(includePreRelease, constants.preRelease)
-            .arg(constants.source).arg(packageSource).argIf(nugetConfigFilePath, constants.configFile)
+        nugetTool.arg(constants.list)
+            .argIf(packageSource === constants.defaultPackageSource , `packageid:${constants.packageId}`)
+            .argIf(packageSource !== constants.defaultPackageSource , `${constants.packageId}`)
+            .argIf(includePreRelease, constants.preRelease)
+            .arg(constants.noninteractive).arg(constants.source).arg(packageSource)
```

Comment:
if tool supports named parameters, we should use that instead of sending 'space separated' data

Reply:
these are named params.